community-containers/dlna/readme.md

```diff
@@ -1,5 +1,11 @@
 ## DLNA server
-This container bundles DLNA server for your Nextcloud files to be accessible by the clients in your local network. Simply run the container and look for a new media server Nextcloud-AIO in your local network.
+This container bundles DLNA server for your Nextcloud files to be accessible by the clients in your local network. Simply run the container and look for a new media server `nextcloud-aio` in your local network.
+
+### Notes
+- This container needs the `nextcloud-aio-nextcloud` container to be up and running to work
+- This is not working on Docker Desktop since it needs `network_mode: host` in order to work correctly
+- This is not working on Mac OS since the `host` networking mode doesn't really share the host's network interfaces in this system
```

Comment:
I think Docker Desktop includes macOS?

Reply:
Those two lines probably mean the same - the `host` mode on Mac OS is unusable for DLNA to work. I'll change that.